crates/bevy_app/src/sub_app.rs

```diff
@@ -332,6 +333,7 @@ impl SubApp {
     }
 
     /// See [`App::get_added_plugins`].
+    #[cfg(feature = "downcast")]
```

Comment:
Leave a TODO to clean this up later please, linking to the PR upstream.

Reply:
Made a `TODO` comment, I'll update it with a link to the PR when it's published too.